new/PRS14029Eraqlis.xml

```diff
@@ -47,17 +47,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                    
                     <persName xml:lang="gez" xml:id="n1">ኤራቅሊስ</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"> Ǝrāqis</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"> <idno>MS GG no. 00044</idno>ʿErāqis</persName>
                     <faith type="EOTC"/>
                     <nationality type="Ethiopia"/>
                     <birth/>
                     <death/>
                     <floruit/>
                 </person>
                 <listRelation>
-                    <relation name="saws:isCopierOf" active="PRS14029Eraqlis" passive="LIT6802SobaAshafaC"/>
+                    <relation name="saws:isCopierOf" active="PRS14029Eraqlis" passive="MS GG no. 00044"/>
```

Comment:
You are right, here the manuscript ID has to be inserted. But the current value is not the manuscript ID. Please check the guidelines or other records to identify what the manuscript ID should be,